.changeset/perfect-wasps-hear.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/backstage-plugin-aws-backend': minor
```

Comment:
This looks to be the incorrect package here.

Reply:
My bad...

```suggestion
'@roadiehq/backstage-plugin-argo-cd-backend': minor
```